src/chage.c

```diff
@@ -247,7 +246,7 @@ print_day_as_date(long day)
 		return;
 	}
 
-	STRFTIME(buf, iflg ? "%Y-%m-%d" : "%b %d, %Y", &tm);
+	STRFTIME(buf, "%Y-%m-%d", &tm);
```

Comment:
Alternatively, I thought of using `"%c"` instead of `"%b %d, %Y"` in non-ISO mode, and adding a boolean flag to DAY_TO_STR to specify if we want ISO mode or locale mode.

Here's what `"%c"` means (from the :

date(1):
```
%c     locale’s date and time (e.g., Thu Mar  3 23:05:25 2005)
```

strftime(3):
```
     %c     The  preferred date and time representation for the current locale.
            (The specific format used in the current locale can be obtained  by
            calling  nl_langinfo(3) with D_T_FMT as an argument for the %c con‐
            version specification, and with ERA_D_T_FMT for the %Ec  conversion
            specification.)   (In  the POSIX locale this is equivalent to %a %b
            %e %H:%M:%S %Y.)
```

But `"%b %d, %Y"` is something that should be removed, since it's not a valid way of printing dates $here, for many values of $here in the world.

Any opinions?

Reply:
Ahh, I hadn't found `"%x"`.  It's more appropriate here, as it only includes date information.

date(1):
```
     %x     locale’s date representation (e.g., 12/31/99)
```
strftime(3):
```
     %x     The  preferred  date  representation for the current locale without
            the time.  (The specific format used in the current locale  can  be
            obtained  by  calling  nl_langinfo(3) with D_FMT as an argument for
            the %x conversion specification, and with  ERA_D_FMT  for  the  %Ex
            conversion specification.)  (In the POSIX locale this is equivalent
            to %m/%d/%y.)
```

Reply:
I'll do that.  We're always in time to switch to always-ISO.  And it's more friendly to use locale for human-readable stuff.